sql/step_95_missing_user_related_fk.sql

```diff
@@ -0,0 +1,550 @@
+ALTER TABLE OH_ADMISSION 
```

Comment:
@mwithi  Shouldn't these changes be made to the load_demo_data.sql file also?

Reply:
Two places where tables are defined? O_o
A recipe for trouble :D

Reply:
The load_demo_data.sql file is typically only a database dump of the database once the demo data is loaded.     That dump includes recreating the tables after dropping them.    Only for very "small" changes is the file modified directly.

Reply:
Yes, we should update the load_demo_data.sql with a new dump.
Anyway, I found some problems in the step_95 that we might adjust before, see my comment here https://openhospital.atlassian.net/browse/OP-1103?focusedCommentId=14998